backend/app/core/yaml_init.py

```
@@ -611,6 +611,76 @@ def scan_and_apply_yaml_directory(
+    """
+    from datetime import datetime
+
+    from app.models.kind import Kind
+
+    # Check if organization KB already exists
+    existing = (
+        db.query(Kind)
+        .filter(
+            Kind.kind == "KnowledgeBase",
+            Kind.namespace == "organization",
+            Kind.is_active == True,
```

Ruff (0.14.14)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)